1.adoc

```diff
@@ -21,6 +21,15 @@ Platforma:: Webový prohlížeč (editor je komponenta použitelná v rámci web
     * co dělal...
 
 == Product statement
+
+Markup editor je webová embedovatelná komponenta pro editaci dokumentů v jazycích LML.
```

Comment:
"v jazycích LML (Lightweight Markup Language)" (ať si to lidi jako já nemusí googlit :) )

Reply:
fixed 9c13096d79ccfc0b873910feac3d9c4772390114